docs/pm_examples/greeter/1.0.0-pretty.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Haha, it's awesome finally seeing a new manifest that's not one of the use-case examples 👍